crates/wash-lib/src/build.rs

```diff
@@ -117,28 +149,30 @@ pub fn build_actor(
             &actor_wasm_path,
             &actor_wasm_path,
         )?;
-    };
+        };
+
+        // If the actor has been configured as WASI Preview2, adapt it from preview1
+        if actor_config.wasm_target == WasmTarget::WasiPreview2 {
```

Comment:
Do we not want to try to adapt actors built with a custom command? If not, I think we should call this out in the docs on `build_command`

Reply:
Good call. Basically, the thought is if you're adding a custom build command you're telling wash that you know what you're doing and not to interfere.

Reply:
For context, one such application of this is using `cargo component build --release` to build a straight up wasip2 component, and in that case no adapting is needed